demoapp/package.json

```diff
@@ -19,15 +19,15 @@
     }
   },
   "dependencies": {
-    "cordova-android": "^7.0.0",
+    "cordova-android": "^7.1.4",
     "cordova-ios": "^5.0.0",
-    "cordova-plugin-appcenter-analytics": "^0.4.1",
-    "cordova-plugin-appcenter-crashes": "^0.4.1",
-    "cordova-plugin-appcenter-push": "^0.4.1",
-    "cordova-plugin-appcenter-shared": "^0.4.1",
+    "cordova-plugin-appcenter-analytics": "file:../cordova-plugin-appcenter-analytics",
```

Comment:
I don't see in the updated release WIKI where this would need to be reverted to NPM external dependency instead of project reference.

Reply:
addressed